### PR TITLE
Added nix pkgs to README

### PR DESCRIPTION
**Pull request by davidv171**
Added nixpkgs link to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,11 +87,11 @@ $ hyperfine --warmup 15 --min-runs 50 \
 Benchmark #1: maim > /dev/null
   Time (mean ± σ):     629.3 ms ±   3.7 ms    [User: 570.7 ms, System: 52.1 ms]
   Range (min … max):   624.8 ms … 646.7 ms    50 runs
- 
+
```
Comment (9ary) [inline README.md:90]: This really shouldn't be part of this pull request, feel free to submit another one for fixing the trailing white space, but please don't include it here.

File: README.md [hunk 2]
```diff
@@ -154,4 +154,5 @@ Summary
   - or install from [crates.io](https://crates.io/crates/shotgun):
     `cargo install shotgun`
 - [Arch Linux](https://www.archlinux.org/packages/?name=shotgun): `pacman -S shotgun`
+- [Nix/NixOS from 20.03 onwards](https://github.com/NixOS/nixpkgs-channels/blob/nixos-20.03/pkgs/tools/graphics/shotgun/default.nix) `nix-env -iA nixpkgs.shotgun`
```
Comment (9ary) [inline README.md]: I think I'd rather point to unstable so the link is always to the most current version of the package, and I don't think it's necessary to specify which nixpkgs releases it's available in. Also, `nix-env` isn't really the recommended practice for "installing" packages on NixOS, I guess just `nixpkgs.shotgun` would be enough to get the point across.
```suggestion
- [Nix/NixOS](https://github.com/NixOS/nixpkgs-channels/blob/nixos-unstable/pkgs/tools/graphics/shotgun/default.nix): `nixpkgs.shotgun`
```

**Review by 9ary — CHANGES_REQUESTED** [review]
Thanks! Please address my comments and this should be good to merge.

**Comment by bb010g** [comment]
I'll probably keep the NUR repo around, but if our packages are in upstream Nixpkgs, we should link that first. Also, I'd very much not like to recommend `nix-env` for installation.

If you give me a few minutes, I'll just patch this up to satisfaction & merge. (You'll still receive contribution credit.)

**Comment by 9ary** [comment]
Maybe add the NUR link as well then, since we don't have one currently.

File: README.md (hunk 1 above)
Comment (Lonami) [reply README.md:90]: I think it's fine to have these minor drive-by changes. We're updating the README, so might as well clean up some tiny things.

Comment (davidv171) [reply README.md:90]: Oh I didn't actually want to include this at all, but nvim refused to not do it, so I got tired of fighting it and committed it.

Comment (Lonami) [reply README.md:90]: You should be able to `git add --patch` to select which hunks to include, so even if there are unwanted changes in a file you can select to ignore them.

**Comment by 9ary** [comment]
Thanks!

**Pull request merged**